### ISSUE TITLE
Aparentemente não submetido.

### ISSUE THREAD

**Issue by monacofj** [courselab/hello-world-SilvioRibeiro-Nusp11836858#2]
Necessita de ajuda?

**Comment by SilvioRibeiro-Nusp11836858** [comment]
Peço desculpas pela demora professor, mas estou quase conseguindo.

**Comment by SilvioRibeiro-Nusp11836858** [comment]
Certo... Depois de muitos problemas para instalar as ferramentas no computador, acabei optando pela ferramenta online. mas aparentemente, preciso de uma autorização para submeter o programa.

**Comment by SilvioRibeiro-Nusp11836858** [comment]
Vou tentar fazer o segundo exercício pra ver se essa situação se repete.
